lib/engine/game/g_1841/step/buy_new_tokens.rb

```diff
@@ -12,6 +12,7 @@ class BuyNewTokens < Engine::Step::Base
 
           def actions(entity)
             return [] unless entity == pending_entity
+            return [] unless entity.corporation?
```

Comment:
This isn't the root of the issue, so probably not the best fix. The root is that the corporation gets set to `pending_entity`.